spec/routing/hq/dashboard_routing_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe 'Hq::Dashboard routing', type: :routing, pending: true do
+
+  specify { expect(get: '/hq/dashboard'). to be_routable }
+
+end
+
+RSpec.describe "Hq::Dashboard redirecting", type: :request, pending: true do
+  before :each do
+    sign_in instance_double(AdminUser)
+  end
+
+  it "root redirects to dashboard" do
+    get '/hq'
+    expect(response).to redirect_to hq_dashboard_path
+  end
+end
```

Comment:
@theB264 Yes, using pending specs looks more professional. Thanks.
I created this spec but when I run it, `sign_in instance_double(AdminUser)` is giving me an error:
`Could not find a valid mapping for #<RSpec::Mocks::InstanceVerifyingDouble:0x307cee0 @name="AdminUser (instance)">`
I did research to fix it but no luck. Do you have any idea how to make this work?

There is some info about this on Devise docs but not very helpful:
`@request.env["devise.mapping"] = Devise.mappings[:user]`